jarviscli/plugins/voice.py

```diff
@@ -14,7 +14,7 @@ def disable_sound(jarvis, s):
 
 
 @plugin('say')
-def say(jarvis, s):
```

Comment:
This change I don't understand. And breaks say-command.

But otherwise great work!

Reply:
Sorry! Might have changed it mistakenly changed it while breaking the code :sweat_smile: I will fix it with a commit!